test/java/org/chromium/net/BidirectionalStreamTest.java

```diff
@@ -1543,24 +1542,28 @@ public void testCronetEngineShutdownAfterStreamCancel() throws Exception {
   @OnlyRunNativeCronet
   @Ignore("https://github.com/envoyproxy/envoy-mobile/issues/1594")
   public void testErrorCodes() throws Exception {
+    // TODO(Augustyniak) cannot find symbol checkSpecificErrorCode(NetError.ERR_ADDRESS_UNREACHABLE"
     // Non-BidirectionalStream specific error codes.
-    checkSpecificErrorCode(NetError.ERR_NAME_NOT_RESOLVED,
-                           NetworkException.ERROR_HOSTNAME_NOT_RESOLVED, false);
-    checkSpecificErrorCode(NetError.ERR_INTERNET_DISCONNECTED,
-                           NetworkException.ERROR_INTERNET_DISCONNECTED, false);
-    checkSpecificErrorCode(NetError.ERR_NETWORK_CHANGED, NetworkException.ERROR_NETWORK_CHANGED,
-                           true);
-    checkSpecificErrorCode(NetError.ERR_CONNECTION_CLOSED, NetworkException.ERROR_CONNECTION_CLOSED,
-                           true);
-    checkSpecificErrorCode(NetError.ERR_CONNECTION_REFUSED,
-                           NetworkException.ERROR_CONNECTION_REFUSED, false);
-    checkSpecificErrorCode(NetError.ERR_CONNECTION_RESET, NetworkException.ERROR_CONNECTION_RESET,
-                           true);
-    checkSpecificErrorCode(NetError.ERR_CONNECTION_TIMED_OUT,
-                           NetworkException.ERROR_CONNECTION_TIMED_OUT, true);
-    checkSpecificErrorCode(NetError.ERR_TIMED_OUT, NetworkException.ERROR_TIMED_OUT, true);
-    checkSpecificErrorCode(NetError.ERR_ADDRESS_UNREACHABLE,
-                           NetworkException.ERROR_ADDRESS_UNREACHABLE, false);
+    // checkSpecificErrorCode(NetError.ERR_NAME_NOT_RESOLVED,
```

Comment:
@RyanTheOptimist I'm good commenting these out if you are.  I hate to have upgrades blocked on it.

Reply:
I think this sounds reasonable, but we should probably file an issue about this so we don't lose track of it. Maybe @colibie has thoughts on how to resolve the missing symbol issues.

Reply:
Turns out the previous impl was using the symbols from the cronet in the android platform. Will fix this with [PR2568](https://github.com/envoyproxy/envoy-mobile/pull/2568)

Reply:
@colibie thank you for the context. I am going to wait with my PR until yours is merged then.

Reply:
You can merge yours, I'll update mine. Still making some changes on it anyway.